.travis.yml

```diff
@@ -18,3 +18,4 @@ script:
   - go env
   - go fmt
   - env GO111MODULE=on go build
+  - ./tests/test_snapshotter.sh
```

Comment:
@ganeshmaharaj you need to install your lvm tools and other dependencies. the travis VM will not have them